pkg/cmd/controller/controller_build.go

```diff
@@ -1268,18 +1277,34 @@ func (o *ControllerBuildOptions) reportStatus(kubeClient kubernetes.Interface, n
 	if pipelineContext == "" {
 		pipelineContext = "jenkins-x"
 	}
-	description := status
+
+	var runningStages []string
```

Comment:
Personally, I'd put that into a function _createStatusDescription_, but up to you.
This change is really independent of Lighthouse, right? This new description would be available with Prow as well, right?

Reply:
It could be used with Prow, in theory, but it would just result in duplicate reporting, since a Prow already does its own reporting controlled through Prow. That said, I’m gonna do a follow up like you describe.